Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8
+
+ENV MODULE_NAME="raddar.main"
+
+WORKDIR /raddar/
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_HOME=/opt/poetry python && \
+    cd /usr/local/bin && \
+    ln -s /opt/poetry/bin/poetry && \
+    poetry config virtualenvs.create false
+
+# Copy poetry.lock* in case it doesn't exist in the repo
+COPY ./pyproject.toml ./poetry.lock* /raddar/
```

Comment:
I wonder if we should specify the full path `/raddar/` every time or instead rely on the `WORKDIR` chosen above

Reply:
I prefer to specify the full path when changing the workdir